spec/features/service_providers_spec.rb

```diff
@@ -39,7 +40,7 @@
       expect(page).to have_content('email')
       expect(page).to have_content(user.teams[0].agency.name)
       expect(page).to have_content('IAL2')
```

Comment:
**comment:** I think this assertion is passing because we say something like "we don't meet IAL2 at this time" in the help text for the level of service question, but it's not accurately checking that the correct option is selected. Do you want to update this assertion to also check for the correct `I18n` content and make sure that still passes?